device/cuda/src/cca/component_connection.cu

```diff
@@ -37,7 +37,7 @@ struct cell_container {
     channel_id* channel1 = nullptr;
     scalar* activation = nullptr;
     scalar* time = nullptr;
-    geometry_id* module_id = nullptr;
+    unsigned int* module_id = nullptr;
```

Comment:
Did I miss something? Why did you remove the `geometry_id` type? :confused: Typedefs are generally good in this type of code.

Reply:
Because I made this code match the "alt_cell" type, which uses links to a module collection to store the geometry information, not the actual geometry ids

Reply:
:thinking: Then let's rename the variable to something like `module_link`. As `module_id` would suggest a different content.